04/index.html

```diff
@@ -1,30 +1,66 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 800px)" href="./styles/tablet.css">
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
-        </ul>
-    </nav>
+
+    <div class="container">
+        <nav class="nav">
+            <label class="nav__label" for="menu-switch">
+                <div class="nav__hamburger">
+                    <span class="hamburger__element"></span>
```

Comment:
Jak tworzysz "nowy" element tj. `hamburger__element` to trzeba zanzaczyć przez `block` gdzie się on znajduje tj. np. dla `<div>` dodać drugą klasę o nazwie `hamburger`.